resources-for-our-team.html

```diff
@@ -0,0 +1,51 @@
+    <br>
+    <h4>Office Hours &amp; Tech Support</h4>
+    <p>English Now!'s office is still open online and by phone - at regularly scheduled hours, Monday through Thursday, 9:00am until 5:00pm; Fridays 9:00am until 1:00pm; and Saturdays 9:30am until 12:00pm.</p>
+    <p>Please note: 1) **Please call or text Ioanna on her mobile phone on Monday and Thursday between 2:00 and 4:00pm ONLY**; 2) if you are having a technical issue outside that time, please <u>text the school phone</u> at <a href="tel:+13017183575">+1.301.718.3575</a>. State succinctly what the issue is, and how urgent it is. We may change the protocol for tech support and will inform you if this changes. Thank you so much for your support!</p>
+    <br>
+    <h4>Online Teaching: Zoom Webinars and Related Materials</h4>
+    <ol>
+        <li>Start here, <strong>a great musical tribute to teaching online</strong> with Zoom, see: <a target="_blank" rel="noopener" href="https://www.youtube.com/watch?v=CCe5PaeAeew&amp;feature=youtu.be">https://www.youtube.com/watch?v=CCe5PaeAeew&amp;feature=youtu.be</a> (thanks, Susan, for sharing this!)</li>
+        <li>Anna Green speaking about <strong>Getting Ready for Online Teaching</strong> (21:56) (Anna re-recorded this as a recap of her presentation), see: <a target="_blank" rel="noopener" href="https://english-now.zoom.us/rec/share/--skPbjA8GJObKOO6gKOCrQsAtjeeaa8hyZP_KcEmk5vxZKuifPhtJjQVLiL1tbT">https://english-now.zoom.us/rec/share/--skPbjA8GJObKOO6gKOCrQsAtjeeaa8hyZP_KcEmk5vxZKuifPhtJjQVLiL1tbT</a> (Zoom Webinar, March 14, 2020)</li>
+        <li>Article on <strong>"Seven Tips on How to Prepare for Teaching Online"</strong>, <em>cited by Anna in her Zoom Webinar</em>, providing a great overview of basic thoughts about online teaching, see: <a href="https://elearningindustry.com/7-tips-prepare-for-teaching-online">https://elearningindustry.com/7-tips-prepare-for-teaching-online</a> (referenced in Zoom Webinar, March 14, 2020)</li>
+        <li>Bryce Gold speaking about <strong>Basic Principles of Online Teaching</strong> (using curriculum, screen sharing, etc.) (52:22), see: <a target="_blank" rel="noopener" href="https://english-now.zoom.us/rec/share/-9EvA7DOtUlLQoXr8h3ld4d_LKTOeaa80CVM8qBezR4EzYt9ZcICUiATDHIHqDuR">https://english-now.zoom.us/rec/share/-9EvA7DOtUlLQoXr8h3ld4d_LKTOeaa80CVM8qBezR4EzYt9ZcICUiATDHIHqDuR</a> (Zoom Webinar, March 14, 2020)</li>
+        <li>Paul Boesen speaking about <strong>Zoom Administration Basics</strong>, including <q>Seven Tips on How to Start a Class</q> (47:32), see: <a target="_blank" rel="noopener" href="https://english-now.zoom.us/rec/play/7pIlfu-srT43T9yW5ASDV6QqW43ue6ms1ChN-6AEmEfjUnVRMAClbrEUMOJaHQspGcpbksU4jFcNVGvm?continueMode=true">https://english-now.zoom.us/rec/play/7pIlfu-srT43T9yW5ASDV6QqW43ue6ms1ChN-6AEmEfjUnVRMAClbrEUMOJaHQspGcpbksU4jFcNVGvm?continueMode=true</a> (Zoom Webinar, March 16, 2020)</li>
```

Comment:
Would recommend using some sort of text (like "here" or "view on Zoom") rather than the full link